spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/repository/RdbmsStreamDefinitionRepositoryTests.java

```diff
@@ -368,6 +369,148 @@ public void findAllPageableDefinitionStringTestDESC() {
 		findAllPageable(pageable, names);
 	}
 
+	// Search Tests
+
+	@Test
+	public void findAllUsingSearchPageablePage2TestDESC() {
+		Sort sort = new Sort(new Sort.Order(Sort.Direction.DESC, "DEFINITION_NAME"), new Sort.Order(Sort.Direction.DESC, "DEFINITION"));
+		Pageable pageable = new PageRequest(1, 2, sort);
+		final SearchPageable searchPageable = new SearchPageable(pageable, "stream");
+		searchPageable.addColumns("DEFINITION_NAME", "DEFINITION");
+
+		String[] names = new String[]{"stream1"};
+		findAllUsingSearchPageable(searchPageable, names);
+	}
+
+	@Test
+	public void findAllUsingSearchPageableDefinitionStringTestDESC() {
+		final Sort sort = new Sort(new Sort.Order(Sort.Direction.DESC, "DEFINITION"));
+		final Pageable pageable = new PageRequest(0, 2, sort);
+		final SearchPageable searchPageable = new SearchPageable(pageable, "eam");
```

Comment:
Should the search criteria for the paging tests be`stream` instead of `eam` for consistency with the TestDesc2?